Dockerfile

```diff
@@ -1,12 +1,37 @@
-FROM thmhoag/steamcmd:latest
+ARG STEAMCMD_VERSION=latest
+ARG AMG_BUILD=latest
+ARG AMG_VERSION=v1.6.57
+FROM drpsychick/steamcmd:$STEAMCMD_VERSION AS base
```

Comment:
what do you think of using steamcmd/steamcmd?

Reply:
needs some more adjustments as it does not create the `steam` user

Reply:
it's based on using `root` user. I'd rather use the existing image for now.
https://github.com/steamcmd/docker/blob/master/dockerfiles/ubuntu-18/Dockerfile

Reply:
right, good point, I hadn't caught that